package.json

```diff
@@ -72,5 +72,9 @@
     "./**/*.{js,jsx,ts,tsx,scss}": [
       "npm run format"
     ]
+  },
+  "volta": {
+    "node": "14.19.1",
```

Comment:
Just a suggestion:
we can use ` "node": ">=14" ` 
instead of having it in two lines.

Reply:
Yeah, good suggestion. But using exactly the same version in development as CI uses is good we are confident that the build will not fail for version issues.

Reply:
What are your thoughts on this?

Reply:
Ok I will make the changes